pkg/convert/byteConverter.go

```diff
@@ -0,0 +1,32 @@
+package convert
+
+import (
+	"encoding/binary"
+	"unicode/utf16"
+)
+
+// Encode uint64 to byte array.
+func Uint64ToByteArrayU8(u64 uint64) []byte {
+	encode := 8
+	b := make([]byte, encode)
+	binary.LittleEndian.PutUint64(b, u64)
+
+	return b
+}
+
+// Decode uint32 to string.
+func EncodeUint32ToUTF8String(numberToEncode uint32) string {
+	//nolint:gomnd
+	buffer := make([]byte, 4)
```

Comment:
same: const + u32NbBytes

Reply:
![image](https://user-images.githubusercontent.com/10938862/205630243-6f483339-a496-4976-8366-9a53997c2d31.png)


Impossible to make and const on same var

Reply:
To delete, not used anymore